src/main/java/com/mycompany/myapp/web/rest/reservation/ReservationResource.java

```diff
@@ -0,0 +1,46 @@
+package com.mycompany.myapp.web.rest.reservation;
+
+import com.codahale.metrics.annotation.Timed;
+import com.mycompany.myapp.service.reservation.ReservationService;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/reservation")
+public class ReservationResource {
+
+    ReservationService reservationService;
+
+    public ReservationResource(ReservationService reservationService){
+        this.reservationService = reservationService;
+    }
+
+    /**
+     * GET  /reserve : make reservation for pool
+     *
+     */
+    @GetMapping("/reserve")
+    @Timed
+    public void activateAccount(@RequestParam(value = "poolId") String poolId, @RequestParam(value = "count") int count) throws Exception {
+        reservationService.reserve(poolId, count);
+    }
+
+    /**
+     * GET  /get : make reservation for pool
+     *
+     */
+    @GetMapping("/get")
+    @Timed
+    public void activateAccount(@RequestParam(value = "poolId") String poolId) throws Exception {
+        reservationService.getAllByPoolId(poolId);
+    }
+
+    /**
+     * GET  /get : make reservation for pool
+     *
+     */
+    @DeleteMapping("/delete")
+    @Timed
+    public void activateAccount(@RequestParam(value = "poolId") Long id) throws Exception {
+        reservationService.deleteReservation(id);
```

Comment:
I am not entirely sure about the method of passing the id, so check it.

Do something like this:
```
public ResponseEntity<Void> activateAccount(...) {
    reservationService.deleteReservation(id);
    return ResponseEntity.ok().headers(HeaderUtil.createAlert( "pools.deleted", "")).build();
}
```

As for the signature i'd do sthing like:
```
@DeleteMapping("/delete/{id}")
@Timed
public void activateAccount(@PathVariable(value = "id") Long id){...}
```
So the small labels in frontend will inform user accordingly.

Also I kindly don't agree with the name of a method. Sounds like copy-paste :)